### ISSUE TITLE
Improve the error message of initialization checker by showing the uninitialized fields

### ISSUE THREAD

**Issue by aosen-xiong** [eisop/checker-framework#622]
The issue is from #621. 

When the initialization checker issues errors related to uninitialized fields, it would be helpful if it could also indicate which specific fields have not been initialized.

**Comment by Ao-senXiong** [comment]
@wmdietl I think right now we do show the uninitialized fields in the error message. See the following example:
```
class Demo {
    String s;
    Object foo;

    Demo() {
        init();
    }

    void init() {}
}
```
Command :  ```java -jar checker/dist/checker.jar -processor nullness FinalClass.java```
Error message:
```
FinalClass.java:5: error: [initialization.fields.uninitialized] the constructor does not initialize fields: s, foo
    Demo() {
    ^
FinalClass.java:6: error: [method.invocation.invalid] call to init() not allowed on the given receiver.
        init();
            ^
  found   : @UnderInitialization Demo
  required: @Initialized Demo
2 errors
```

**Comment by wmdietl** [comment]
Try the following changed constructor:

````
    Demo() {
        s = "";
        init();
        foo = "";
    }
````

**Comment by Ao-senXiong** [comment]
Great! Thanks!
